internal/function/commit_stats.go

```diff
@@ -96,20 +99,39 @@ func (f *CommitStats) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 
 	r, err := f.resolveRepo(ctx, row)
 	if err != nil {
-		return nil, err
+		ctx.Warn(0, "commit_stats: unable to resolve repository")
```

Comment:
Lets wrap this by `logWarn(ctx, ...)` or sth. like this

Reply:
Just to avoid the `0, `?
In this case the messages are the same, in the rest of the errors it's not. Add the function just for this case doesn't seem worth it imho

Reply:
I just thought to have a function for logs and session warnings

Reply:
yup, the thing is the log and the warnings are not the same messages

if you have

ctx.Warn(0, "something", with, args)
logrus.WithFields(some fields).Error("different message")

you will need something like:

logWarn(ctx, "message for warn", []interface{}{args, for, warn}, "message for log", logrus.Fields{args, for, log})

And it kinda defeats the purpose